about.html

```diff
@@ -0,0 +1,144 @@
+This year, we hope to expand our invitation list including organizations and individuals.
+ </article>
+ <article class="about-discription1">
+The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together
+ </article>
+</div>
+ <div class="contact-us">
+<p class="raw-text"> 
+Please contact us per Email for nay further questions about Dbn Hiphop Fest! <br>
+info@hiphopfest.com
+</p>
+ </div>
+  </div>
+  <div class="logo-compertition">
+       <h2 class="inter-act3">For underground artist submit your logo here</h2> 
+
+<hr class="horizontal-main-line">
+  <p class="winner-circle">
+      This years Edition you can stand a chance to win $5000,enter here: info@hiphopfest.com
+  </p>
+ <div class="contact-us">
+ <a class="go1" href="#"><span class="ogo1">Techmoves</a>
+
+ </div>
+  </div>
+ <h2 class="inter-act2">Peep the previous shows</h2> 
+<hr class="horizontal-main-line">
+ <p class="raw-text">
+ This is before covid shows that we had in Moses Mabhida Stadium and KingSpark Stadium
+ </p>
+
+ <div class="last-shows">
+     <div class="pic-overlay">
+ <div class="mabhida">
+     <img class="mabhida1" src="images/2019.png" alt="2019-show">
+     <h4 class="pic-year1">2019</h4>
+     <p class="vanue1">Moses Mabhida Stadium</p>
+ </div>
+ <div class="spark">
+     <img class="spark1" src="images/2022.png" alt="2020-show">
+     <h4 class="pic-year"> 2020</h4>
+     <p class="vanue">Kings Spark Stadium</p>
+ </div>
+     </div>
+ </div>
+</section>
+<footer id="sponsers1">
+     <div class="pantners">
+  <h3 class="inter-act4">Sponsers</h3>
+ <hr class="horizontal-main-line">
+<div class="shows2019">
+
+    
+</div>
+<div class="shows2020">
+    <div class="Sponsr1">Castle Lite</div>
+    <div class="Sponsr2">Budwiser</div>
+    <div class="Sponsr3">MTV</div> <br>
+    <div class="Sponsr4">SPORTIFY</div>
+
+</div>
+
+    
+</div>
+</footer>
+</div>
+<div class="bottom-footer22">
+<div class="bot">
+<a class="slogo" href="#">Techmoves</a>
+<p class="closing-footer1">
+        2023 Gurban hip hop fest. Some Rights Reserved. </p>
+ <p class="chana-bottom1"> 
+달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다. 자세한 이용기능 내용과 허락조건은 http://techmoves.org/licences/by/2.0/kr 를 보세요.
+ </p>
+</div>
+</div>
```

Comment:
- [ ] Kindly add the missing footer element in your about.html page. 👍

Reply:
Resolved

Reply:
resolved